### PR TITLE
Removes a pixel from the DA.

### PR DESCRIPTION
**Pull request by francinum**
Invert of #10192

### CODE REVIEW

**Comment by xxalpha** [comment]
but why though

File: tgstation.dme [hunk 1]
```diff
@@ -6,6 +6,59 @@
 
 // BEGIN_FILE_DIR
 #define FILE_DIR .
+#define FILE_DIR "code"
```
Comment (RemieRichards) [inline tgstation.dme]: No.
no FILE_DIR crap.

**Comment by francinum** [comment]
https://gyazo.com/4cbddd8f1f230e4108bb6eb27215be34

File: tgstation.dme (hunk 1 above)
Comment (francinum) [inline tgstation.dme]: Goddamnit.

**Comment by Chocobro** [comment]
@Iamgoofball because see the other PR

**Comment by Chocobro** [comment]
@xxalpha because see the other PR

**Comment by Jalleo** [comment]
Consistency tag needed

**Pull request merged**